README.md

```diff
@@ -23,6 +23,14 @@
 
 Scribe applications are available on [iOS](https://github.com/scribe-org/Scribe-iOS), [Android](https://github.com/scribe-org/Scribe-Android) (WIP) and [Desktop](https://github.com/scribe-org/Scribe-Desktop) (planned).
 
+# Architecture Diagrams 
```

Comment:
We can simply remove this here too as that would create a header for a new section.
The idea is to just make the below blurb part of the introductory section.